zha/zigbee/cluster_handlers/__init__.py

```diff
@@ -455,9 +455,10 @@ async def async_initialize(self, from_cache: bool) -> None:
             return
 
         self.debug("initializing cluster handler: from_cache: %s", from_cache)
-        cached = [a for a, cached in self.ZCL_INIT_ATTRS.items() if cached]
-        uncached = [a for a, cached in self.ZCL_INIT_ATTRS.items() if not cached]
-        uncached.extend([cfg["attr"] for cfg in self.REPORT_CONFIG])
+        cache_config = self.ZCL_INIT_ATTRS.copy()
+        cache_config |= {cfg["attr"]: False for cfg in self.REPORT_CONFIG}
+        cached = [a for a, cached in cache_config.items() if cached]
+        uncached = [a for a, cached in cache_config.items() if not cached]
```

Comment:
Actually, can we not remove this here, as this might change in a different way in the future and instead just remove it from `cluster_handler.__dict__[zha_const.ZCL_INIT_ATTRS]` where the `REPORT_CONFIG` is modified in `discovery`?